ietf/help/views.py

```diff
@@ -1,30 +1,20 @@
 # Copyright The IETF Trust 2007, All Rights Reserved
 
-from django.shortcuts import get_object_or_404, render
-
 import debug                            # pyflakes:ignore
 
-from ietf.doc.models import State, StateType
 from ietf.name.models import StreamName
+from django.shortcuts import redirect
+
+# These are just redirects to the new URLs under /doc; can probably go away eventually.
 
 def state_index(request):
-    types = StateType.objects.all()
-    names = [ type.slug for type in types ]
-    for type in types:
-        if "-" in type.slug and type.slug.split('-',1)[0] in names:
-            type.stategroups = None
-        else:
-            groups = StateType.objects.filter(slug__startswith=type.slug)
-            type.stategroups =  [ g.slug[len(type.slug)+1:] for g in groups if not g == type ] or ""
-                
-    return render(request, 'help/state_index.html', {"types": types})
+    return redirect('/doc/help/state/', permanent = True)
 
 def state(request, doc, type=None):
     if type:
         streams = [ s.slug for s in StreamName.objects.all() ]
         if type in streams:
             type = "stream-%s" % type
     slug = "%s-%s" % (doc,type) if type else doc
```

Comment:
else doc.type_id maybe?

Reply:
No, in this context `doc` is a simple string variable that comes from a portion of the URL. Maybe that's a poor choice of variable name (preexisting).